client/webpack.config.js

```diff
@@ -97,7 +97,7 @@ let buildconfig = {
                     loader: "file-loader",
                     options: {
                         outputPath: "assets",
-                        publicPath: "/static/scripts/bundled/assets/"
+                        publicPath: "../scripts/bundled/assets/"
```

Comment:
```suggestion
                        publicPath: path.join(__dirname, "../scripts/bundled/assets/")
```

Reply:
Unless I misread the suggestion, this will not work.  This defeats the purpose of it being relative and will generate a full path like `/Volumes/work_cs/galaxy/static/style`

Reply:
I saw this syntax used in the rest of the file, but feel free to disregard my suggestion.

Reply:
Right, so for actually resolving paths for consumption when building we do want to use the concrete paths.  The publicPath is what actually gets injected into the resultant bundle for resolving resources as deployed, so we won't want the full thing.